Makefile

```diff
@@ -26,6 +26,7 @@ GOFMT_CMD := gofmt -s -l
 GOVET_CMD := go tool vet
 CI_HOST_TARGETS ?= "host-unit-test host-integ-test host-build-docker-image"
 SYSTEM_TESTS_TO_RUN ?= "00SSH|Basic|Network|Policy|TestTrigger|ACIM|Netprofile"
+K8S_SYSTEM_TESTS_TO_RUN ?= "00SSH|TestBasic|TestNetwork|TestPolicy"
```

Comment:
Can we avoid ``Test`` prefix like in ```SYSTEM_TESTS_TO_RUN``` ?

Reply:
@rchirakk: Sure, I can update this to remove the prefix. I've kept it as it was previously to avoid any potential issues.